src/main/resources/normal_event_skip_list.txt

```diff
@@ -1,5 +0,0 @@
-8863008
-8868134
-8863674
-180687
-9652264
```

Comment:
This is deleted because it is added manually into the Jenkins orthoinference directory?

Reply:
Yes and I wanted to enforce that the user should _provide_ the skiplist, rather than having it sit statically. Last release I nearly forgot to update the skiplist (it was different than other releases) and i realized that nothing would have prevented orthoinference from proceeding if I hadn't reemembered -- so this is an attempt at solving that pitfall